browser/resources/brave_new_tab_page_refresh/components/app.style.ts

```diff
@@ -164,51 +164,28 @@ export const style = scoped.css`
     --widget-height: 128px;
     --widget-min-width: 380px;
     --widget-max-width: 512px;
-    --widget-count: 0;
     --widget-gap: 16px;
-    --widget-gap-total:
-      calc((var(--widget-count) - 1) * var(--widget-gap));
-    --widget-available-width: calc(
-      100cqi / var(--widget-count) -
-      var(--widget-gap-total) / var(--widget-count));
-    --widget-flex-basis: max(
-      var(--widget-min-width),
-      min(var(--widget-max-width), var(--widget-available-width)));
 
     anchor-name: --ntp-widget-container;
 
-    align-self: center;
-    flex: 0 0 var(--widget-height);
+    align-self: stretch;
+    flex: 0 0 auto;
+    min-height: var(--widget-height);
 
-    display: flex;
+    display: grid;
```

Comment:
The widget container has been updated to use grid layout which allows get rid of the width calculations above.